spec/fastsync/vdd/fastsyncL2.tla

```diff
@@ -110,7 +116,9 @@ VARIABLES
     peerHeights,                            \* map of peer ids to its (stated) height
     blockStore,                             \* map of heights to (received) blocks
     receivedBlocks,                         \* map of heights to peer that has sent us the block (stored in blockStore)
-    pendingBlocks                           \* map of heights to peer to which block request has been sent
+    pendingBlocks,                          \* map of heights to peer to which block request has been sent
+    syncHeight,                             \* height at the point syncTimeout was triggered last time 
+    syncedBlocks                            \* number of blocks synced since last syncTimeout
```

Comment:
```suggestion
    syncedBlocks                            \* number of blocks synced since last syncTimeout (If it is 0 when the next timeout occurs, then Fastsync terminates)
```